src/components/transactions/message-view/ClaimRewardsTxDetails.vue

```diff
@@ -1,16 +1,13 @@
 <template>
   <div class="tx__content">
-    <TransactionIcon
-      :transaction-group="transaction.group"
-      :transaction-type="type"
-    />
-    <div v-if="getValidators && getValidators.length === 1">
+    <TransactionIcon :transaction-type="type" />
+    <template v-if="getValidators && getValidators.length === 1">
       <div class="tx__content__left">
         <h3>{{ caption }}</h3>
         <div class="multi-claim-reward-row">
-          <span>Rewards from&nbsp;</span>
+          <span>Rewards from</span>&nbsp;&nbsp;
```

Comment:
why 2?

Reply:
Fast & dirty solution to add space (weird, was needed only in that place). I'm going to replace it by proper css!